docs/src/main/sphinx/functions/math.md

```diff
@@ -228,10 +228,19 @@ Returns the base-`radix` representation of `x`.
 
 ## Statistical functions
 
+:::{function} cosine_distance(array(double), array(double)) -> double
```

Comment:
Technically, this is not a statistical function. It would be more appropriate to classify it as geometric or linear algebra function.

Reply:
Do we have docs for cosine_similarity? (for the dense vector version, not the one below)

Reply:
added now